version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.4.0-SNAPSHOT"
```

Comment:
Maybe. I haven't looked at it. Please feel free to fix if you think it is important.